src/commands/secret/ISecretContext.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { Secret } from "@azure/arm-appcontainers";
+import type { ExecuteActivityContext } from "@microsoft/vscode-azext-utils";
+import type { IContainerAppContext } from "../IContainerAppContext";
+
+export interface ISecretContext extends IContainerAppContext, ExecuteActivityContext {
+    // New
+    secretName?: string;
```

Comment:
I think it would be easier to just name the variable names `newSecretName` and `newSecretValue`.